auditlog/models/http_request.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+from openerp.http import request
+
+
+class AuditlogHTTPRequest(models.Model):
+    _name = 'auditlog.http.request'
+    _description = u"Auditlog - HTTP request log"
+    _order = "create_date DESC"
+    _rec_name = 'display_name'
+
+    display_name = fields.Char(u"Name", compute="_display_name")
```

Comment:
What's better in this scenario? `name_get` seems to be less elegant if you still keep the computed `display_name` field. If you get rid of `display_name` and compute directly in `name_get` that has worse performance since it will be computed on every view load that has this model defined.

```python
class AuditlogHTTPRequest(models.Model):

    ...

    display_name = fields.Char(u"Name", compute="_compute_display_name")

    @api.multi
    @api.depends('create_date', 'name')
    def _compute_display_name(self):
        for httprequest in self:
            create_date = fields.Datetime.from_string(httprequest.create_date)
            tz_create_date = fields.Datetime.context_timestamp(
                httprequest, create_date)
            httprequest.display_name = u"%s (%s)" % (
                httprequest.name or '?',
                fields.Datetime.to_string(tz_create_date))

    @api.multi
    def name_get(self):
        res = []
        for request in self:
            res += [(request.id, request.display_name)]
        return res
```